pkg/evacuation/evacuation.go

```diff
@@ -477,3 +580,12 @@ func isAttachableOnOtherNode(pv *corev1.PersistentVolume, currentNodeName string
 
 	return false
 }
+
+// PVCorPVEventf records an event, either for the PVC or for the PV, if no PVC is currently bound.
+func PVCorPVEventf(recorder record.EventRecorder, pv *corev1.PersistentVolume, eventtype, reason, message string, args ...interface{}) {
```

Comment:
Shouldn't it strictly be so?

```suggestion
// PVCOrPVEventf records an event, either for the PVC or for the PV, if no PVC is currently bound.
func PVCOrPVEventf(recorder record.EventRecorder, pv *corev1.PersistentVolume, eventtype, reason, message string, args ...interface{}) {
```

Aside: I prefer `PvcOrPvEventf` or even better `pvc_or_pv_eventf`, but we're not going to change the Go world now.